### ISSUE TITLE
Plot_bgcolor and Paper_bgcolor

### ISSUE THREAD

**Issue by fdrcslv** [plotly/documentation#1408]
On the python reference page the layout properties paper_bgcolor and  plot_bgcolor don't mention "background" in their description, so it's kinda difficult to find them if you simply search for "background" (like most would do).

The defaults seem outdated too, since for plot_bgcolor the new default is rgb(229, 236, 246)

### **paper_bgcolor** (color) 
default: "#fff" 
Sets the color of paper where the graph is drawn.

### **plot_bgcolor (color)** 
default: "#fff" 
Sets the color of plotting area in-between x and y axes.

Thank you for your amazing work

**Comment by sebastianrosado** [comment]
Just stumbled across the same thing. @chriddyp is there a way we can help make those minor improvements other than just flagging them? Thank you.

**Comment by chriddyp** [comment]
@sebastianrosado yes indeed! You can make a PR into plotly.js's schema. The schema is the source of truth for all of the reference docs. Here: https://github.com/plotly/plotly.js/blob/master/src/plots/layout_attributes.js
